lib/callbacks/AccessTokenRequest.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const rp = require('request-promise-native');
+const STBase = require("../STBase");
+const uuid = require('uuid/v4');
+
+module.exports = class AccessTokenRequest extends STBase {
+
+  constructor(clientId, clientSecret, requestId) {
+    super('accessTokenRequest', requestId ? requestId : uuid());
```

Comment:
do "request || uuid()" instead.

Reply:
Thanks! I've been missing that `?:` sugar from Groovy :-)